### ISSUE TITLE
Is this odd Python specific format of encoding texts intentional?

### ISSUE THREAD

**Issue by PiotrCzapla** [facebookresearch/MLDoc#5]
The dataset format seems to be changed so that it is not compatible with Laser repository, and one change seems like an unintentional side effect. Namely, the articles texts are encoded as `repr` representation of python byte string which makes the data set very python specific and hard to parse even in python.

Please have a look at this laser issue: https://github.com/facebookresearch/LASER/issues/39 for examples of how the encoding look like.

**Comment by PiotrCzapla** [comment]
The issue is caused by this line:
https://github.com/facebookresearch/MLDoc/blob/master/generate_documents.py#L62
```
output_f.write(
    '{}{}{}\n'.format(topics[0], delim_str, doc.encode('utf-8'))
)
```
I have a feeling the intention was to encode everything as utf-8, if so the line should be as follows:
```
output_f.write(
   ('{}{}{}\n'.format(topics[0], delim_str, doc)).encode('utf-8')
)
```

**Comment by anassalamah** [comment]
I actually didn't need to do that even though I downloaded the Reuters dataset on April 11th. What I found out was that to be able to run mldoc.sh without the "ValueError; could not convert string to float: '0CAT'" I needed to change that code to: 
`output_f.write(
                    '{}{}{}\n'.format(topics[0][0], delim_str, doc)
                )`
